docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-space-manager.md

```diff
@@ -5,10 +5,12 @@
 ```shell
 kind create cluster --name sm-mgt
 KUBECONFIG=~/.kube/config kubectl config rename-context kind-sm-mgt sm-mgt
-export SM_CONFIG=~/.kube/config
-export SM_CONTEXT=sm-mgt
+SM_CONFIG=~/.kube/config
```

Comment:
I am no bash expert so I am just asking - we don't need to export the SM_CONFIG in case the generated script calls another script?  Or are all the scripts being converted to accept SM_CONFIG as parameter?

Reply:
This PR is changing the KubeStellar scripts to not read the `SM_CONFIG` or `SM_CONTEXT` environment variables. This is part of fixing #1438 : getting the kube go client library to read, when running in a pod, the pod's credentials.